torch/csrc/distributed/c10d/init.cpp

```diff
@@ -520,7 +528,10 @@ Example::
       )")
       .def(py::init<>());
 
-  shared_ptr_class_<::c10d::TCPStore>(module, "TCPStore", store,
+  shared_ptr_class_<::c10d::TCPStore>(
+      module,
+      "TCPStore",
+      store,
```

Comment:
nit: Usually try to avoid formatting changes since it makes the PR harder to review. You can always send a separate PR with the formatting changes.

Reply:
Yeah, realized that. Will be more careful next time.